chrome_extensions/Pedrokeys/hiq_hotkeys.js

```diff
@@ -30,15 +30,29 @@ const BANNER_CLASS = 'hiq-banner';
 const HANGUP_BUTTON = '.end-contact';
 const HANGUP_CONFIRM_BUTTON = '.confirm-end-contact';
 const SUBMIT_BUTTON = '.save-close-acw-submit';
-const PREFIX = 'ctrl';
-const FRIENDLY_PREFIX = 'Ctrl';
+var PREFIX = "";
+var FRIENDLY_PREFIX = "";
 
 class HiqHotkeys{
   constructor(){
     this.helpText = [];
+    this.isMac();
     this.bindKeys();
   }
 
+  isMac(){
+    if (navigator.platform.indexOf('Mac') > -1){
+      console.log("Mac");
+      PREFIX = 'ctrl';
```

Comment:
down here just say `this.prefix = 'ctrl'` and `this.friendlyPrefix = 'Ctrl'`. Then use the instance variables below.